### PR TITLE
Added scanning settings to documentation

### PR DESCRIPTION
**Pull request by j0shbregman**
Updated documentation to include hardening of DisableArchiveScanning, DisableScanningNetworkFiles settings..

### CODE REVIEW

**Comment by j0shbregman** [comment]
@denisebmsft Please just wait for me to confirm.

File: microsoft-365/security/defender-endpoint/prevent-changes-to-security-settings-with-tamper-protection.md [hunk 1]
```diff
@@ -60,6 +60,8 @@ During some kinds of cyber attacks, bad actors try to disable security features,
 - Removing security intelligence updates
 - Disabling automatic actions on detected threats
 - Suppressing notifications in the Windows Security app
+- Disabling scanning of archives and network files
+
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/prevent-changes-to-security-settings-with-tamper-protection.md:64]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @j0shbregman, kindly commit the suggested change to remove the extra blank line. Many thanks.

**Comment by j0shbregman** [comment]
@denisebmsft This is good to go.

**Review by denisebmsft — APPROVED** [review]
Thank you @j0shbregman @JohanFreelancer9

**Pull request merged**